src/main/java/com/mraof/minestuck/entity/consort/ConsortEntity.java

```diff
@@ -141,7 +145,20 @@ protected InteractionResult mobInteract(Player player, InteractionHand hand)
 	{
 		if(this.isAlive() && !player.isShiftKeyDown() && eventTimer < 0)
 		{
-			if(!level().isClientSide && player instanceof ServerPlayer serverPlayer)
+			Dialogue dialogue = getDialogue();
+			
+			//TODO there is server side data that needs to be made accessible client side
```

Comment:
One option that I think could work great is to send a server-to-client packet with the necessary dialogue info, which then displays the screen from there.

The alternative would be to try and sync the entity dialogue whenever it changes to nearby players, and have dialogue start directly at the client-side. I think that's the worse option since it's slightly harder to sync the right data, and harder to add extra server-side logic to dialogue start, even if it means that the dialogue screen can pop up earlier without a back-and-forth with the server-side.